packages/nextjs/src/index.types.ts

```diff
@@ -40,6 +40,8 @@ export declare const metrics: typeof clientSdk.metrics & typeof serverSdk.metric
 
 export { withSentryConfig } from './config';
 
+export type { SentryBuildOptions } from './config/types';
```

Comment:
Let's do 
```
export type { SentryBuildOptions } from './types'
``` 

inside `config/index.ts` instead. 

`index.types.ts` already exports everything from `config/index.ts`.

Reply:
here you go